PocketKit/Sources/Textile/Style/Style+Definitions.swift

```diff
@@ -71,19 +71,21 @@ public extension Style {
     }
 
     struct Display {
-        let medium = Medium()
-        let regular = Regular()
+        public let medium = Medium()
+        public let regular = Regular()
```

Comment:
why did we make these `public`

Reply:
I think David did this to keep consistency with the other structs in this file.

Reply:
if we remove it, do we get an error? I lend towards only using public when necessary as opposed to exposing just for consistency sake. what do you think?